src/monocle_apptrace/wrap_common.py

```diff
@@ -427,6 +434,69 @@ def update_span_from_llm_response(response, span: Span, instance):
             token_usage = None
 
 
+def update_events_for_inference_span(response, span, args):
```

Comment:
this method is long and could be modelled on the lines of vector store configuration

Reply:
It would be good to remove any framework references from this file. The target would be to move everything to meta model

Reply:
> this method is long and could be modelled on the lines of vector store configuration

done

Reply:
> It would be good to remove any framework references from this file. The target would be to move everything to meta model

can we do this in separate ticket ? @kshitiz-okahu